tests/proc.rs

```diff
@@ -48,12 +61,15 @@ fn test_pio_proc2() {
 #[test]
 fn test_pio_proc_size() {
     // Inline constant size
-    pio_proc::pio!(32, "label:\njmp label\n");
-    // Constant variable
-    const PROGRAM_SIZE: usize = 32;
-    pio_proc::pio!(PROGRAM_SIZE, "label:\njmp label\n");
-    // Expression
-    pio_proc::pio!(10 + 20, "label:\njmp label\n");
-    // Constant from another crate
-    pio_proc::pio!(pio::RP2040_MAX_PROGRAM_SIZE, "label:\njmp label\n");
+    pio_proc::pio_asm!("label:", "jmp label", options(max_program_size = 32));
+
+    // TODO: Not supported anymore, important?
```

Comment:
Given the max program size is a function of the chip being used, it's a shame we can't use a constant here.

Reply:
I think this should still be supported.

Reply:
> It is? The datasheet says it's defined to be 32 instructions.

The RP2040 datasheet says that, but Raspberry Pi "intends to sell a billion chips in the RP2 family" so there will be others.

But this is fine for now as long as we know what to do when RP2350 or whatever comes along with more of everything.

Reply:
Okey, then I'll look at adding it back

Reply:
Fixed!